packages/generators/__tests__/init-generator.test.ts

```diff
@@ -1,7 +1,7 @@
 import { join } from 'path';
 import { run } from 'yeoman-test';
+const assert = require('yeoman-assert');
```

Comment:
Use the `import`. You are under TypeScript here

Reply:
Tried it but it wasn't working with this package. :(

Reply:
That's probably because you didn't install the types